tests/functiontest/conftest.py

```diff
@@ -809,8 +800,7 @@ def runtestcase(testcase):
     if tc_ignore_test:
         pytest.skip("Test case has 'ignore_test' set")
         return
-    if six.PY2 and tc_ignore_python_version == 2 or \
-            six.PY3 and tc_ignore_python_version == 3:
+    if tc_ignore_python_version == 3:
```

Comment:
Does this make sense now?

Reply:
As long as we support the "ignore_python_version" attribute in the testcase, yes.
However, support for that attribute can be removed now, and I have done that. It was used in testcase invalidresponseerror.yaml, where I deleted the testcases that used it.